### PR TITLE
Add web accessibility

### PR DESCRIPTION
**Pull request by kene-creator**
### Report for web accessibility

- Page Title (Changed page title to "Portfolio home page")
- Image text alternative (Added descriptive alt text to img elements)
- Text heading (No issues found)
- Colour contrast ( Changed font colour of the  font in the work section to be more visible)
- Interaction ( Added label for input elements but set their display to none)
- Moving content (No issues found)
- Multimedia (No issues found)
- Basic structure of the page (No issues found)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Kanit:wght@400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;0,500;0,600;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio home page</title>
```
Comment (kene-creator) [inline index.html:16]: Change title name to "Portfolio home page"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+            <a href="twitter.com">
+              <svg
+                width="20"
+                height="25"
+                viewBox="0 0 20 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-labelledby="twitter"
+              >
+                <title id="twitter">Twitter Logo</title>
```
Comment (kene-creator) [inline index.html:69]: Added title to svg image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,613 @@
...
+            <a href="linkedin.com">
+              <svg
+                width="20"
+                height="28"
+                viewBox="0 0 23 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+                aria-labelledby="linkedin"
+              >
+                <title id="linkedin">linkedin Logo</title>
```
Comment (kene-creator) [inline index.html:87]: Added title for SVG image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,613 @@
...
+          </li>
+        </ul>
+      </div>
+    </header>
+
+    <section class="work-section">
+      <div class="section-img card">
+        <div
+          class="diplay card__side card__side-front card__side-front-4"
+          title="portfolio website example"
```
Comment (kene-creator) [inline index.html:165]: Added image title to div elemet

File: index.html [hunk 5]
```diff
@@ -0,0 +1,613 @@
...
+            <div class="card__price-box">
+              <p class="card__price-only">Only</p>
+              <p class="card__price-value">$200</p>
+            </div>
+            <a href="#" class="btn-card">Book now!</a>
+          </div>
+        </div>
+        <img
+          src="img/Snapshoot-Portfolio.png"
+          alt="portfolio website example"
```
Comment (kene-creator) [inline index.html:180]: Added alt text for better description to the img element

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @kene-creator,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Professional PR message and report. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**